Assets/Scripts/Controllers/Input Output/MouseController.cs

```diff
@@ -591,7 +591,7 @@ private void ShowFurnitureSpriteAtTile(string furnitureType, Tile t)
 
         Furniture proto = PrototypeManager.Furniture.Get(furnitureType);
 
-        go.transform.position = new Vector3(t.X + ((proto.Width - 1) / 2f), t.Y + ((proto.Height - 1) / 2f), WorldController.Instance.cameraController.CurrentLayer);
+        go.transform.position = new Vector3(t.X + ((sr.sprite.pivot.x / sr.sprite.pixelsPerUnit) - 0.5f), t.Y + ((sr.sprite.pivot.y / sr.sprite.pixelsPerUnit) - 0.5f), WorldController.Instance.cameraController.CurrentLayer);
```

Comment:
It looks good. No code duplication is good, specially on this calculations that might change, and are easy to get wrong on copy/paste.